Mathlib/GroupTheory/GroupAction/Quotient.lean

```diff
@@ -401,3 +402,29 @@ theorem quotientCenterEmbedding_apply {S : Set G} (hS : closure S = ⊤) (g : G)
 #align subgroup.quotient_center_embedding_apply Subgroup.quotientCenterEmbedding_apply
 
 end Subgroup
+
+section conjClasses
+
+instance instInfiniteProdSubtypeCommute {M : Type _} [Mul M] [Infinite M] :
+    Infinite { p : M × M // _root_.Commute p.1 p.2 } :=
```

Comment:
Does this clash without the `_root_`?

Reply:
Yes, with Function.Commute which is open

Reply:
Presumably `Function.Commute` should just be protected, because otherwise it's always going to be an ambiguous overload with `open Function`. Can you make that change please?

Reply:
Might it even make sense to delete `Function.Commute`? Regardless, this refactor will probably touch many files. Do you mind if I do it in a separate PR?

Reply:
https://github.com/leanprover-community/mathlib4/pull/6456